src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,53 @@
+package racingcar.domain;
+
+public class Car {
+
+    private static final int MIN_LENGTH = 1;
+    private static final int MAX_LENGTH = 5;
+    private static final int CAN_MOVE_CONDITION = 4;
+    private static final String NAME_LENGTH_EXCEPTION = "자동차 이름은 %d ~ %d글자만 가능합니다.";
+    private static final String NAME_LETTER_EXCEPTION = "자동차 이름에 특수문자나 숫자는 포함될 수 없습니다.";
+
+    private final String name;
+    private int moveCount;
+
+    public Car(String name) {
+        name = name.trim();
+        validateLength(name);
+        validateLetter(name);
```

Comment:
유효성을 검증하는 부분이 많아진다면 코드가 길어질 것 같은데 validateName과 같은 메서드를 작성해서 그 안에서 길이와 문자에 대한 유효성 검증하는 방식은 어떨까요?
```suggestion
public Car(String name) {
        name = name.trim();
        validateName(name);
        this.name = name;
        this.moveCount = 0;
}

private void validateName(String name) {
        validateLength(name);
        validateLetter(name);
}
```

Reply:
현재 가짓수가 많지 않아 따로 분리해주지 않았는데, 변경 가능성을 고려해 분리해주는 것이 더욱 관리하기 좋을 것 같네요!!👍